documentation-site/components/yard/code-box.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from 'react';
+import {useStyletron} from 'baseui';
+
+const CodeBox = ({children}: {children: React.ReactNode}) => {
+  const [css, theme] = useStyletron();
+  const isLight = theme.name === 'light-theme';
```

Comment:
I think we should check for if the name starts with `light-theme`, as the user can toggle the font too. I think we call that `light-theme-move`